composer.json

```diff
@@ -17,6 +17,7 @@
         "symfony/console": "5.2.*",
         "symfony/dotenv": "5.2.*",
         "symfony/framework-bundle": "^5.4",
+        "symfony/http-client": "^5.4",
```

Comment:
This is already a requirement of `wikimedia/toolforge-bundle` so no need to add it separately.